mysql2.gemspec

```diff
@@ -13,9 +13,4 @@ Gem::Specification.new do |s|
 
   s.files = `git ls-files README.md CHANGELOG.md LICENSE ext lib support`.split
   s.test_files = `git ls-files spec examples`.split
-
-  # tests
-  s.add_development_dependency 'rake-compiler', '~> 0.9.5'
-  s.add_development_dependency 'rake', '~> 0.9.3'
-  s.add_development_dependency 'rspec', '~> 2.8.0'
```

Comment:
The gemspec is still the place where runtime dependencies are listed, including for Bundler. I'm on the fence for the internal dependencies, but leaning towards gemspec for convention.

The version bumps look fine to me - newer rake, rake-compiler, and of course newer rspec since you've ported the tests (awesome!).